packages/workspace/src/generators/move/lib/update-eslintrc-json.ts

```diff
@@ -16,10 +16,23 @@ interface PartialEsLintrcOverride {
 }
 
 interface PartialEsLintRcJson {
-  extends: string;
+  extends: string | string[];
   overrides?: PartialEsLintrcOverride[];
 }
 
+function offsetFilePath(
+  project: ProjectConfiguration,
+  pathToFile: string,
+  offset: string
+): string {
+  if (!pathToFile.startsWith('..')) {
```

Comment:
This should probably handle `./.eslintrc-base` or something where the extension starts with `./`

Reply:
Perhaps the comment is misleading, but in reality `./something.ext` points to the same folder so there is nothing that should be changed. The only change needed is with `../../../path/outside/of/project`. The unit test covers this scenario.

Reply:
Ah yes, of course :facepalm: